.github/workflows/linters.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Setup Webhint
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/223378606-21712650-e744-4a17-b026-0f54ec7577df.png)

- [ ] It would be nice if you put your **name** instead of **Author 1** to make your readme file descriptive and professional :blush: